test/splicerTest.js

```diff
@@ -308,5 +308,54 @@ describe('Tag recombination', () => {
     ],
   }));
 
+  it('pick local', () => test(
+    '02-multilingual-alltags',
+    {
+      lng: 'asdf',
+      rlang: 'local',
+      keepUncompressed: true,
+      map: {},
+    },
+    {
+      layers: [
```

Comment:
I might be missing something here, but I don't understand where the actual expectation to pick the local value ("Vancouver") is shown? This whole object is supposedly the 'expected' bit... where do we get the resulting language selection? 

I'm really confused.

Reply:
The expectation is that nothing is touch. Whatever is in name before is still in name after.